docker/README.md

```diff
@@ -85,14 +85,18 @@ Build a full image to your local machine
 docker build .
 ```
 
+**Note:** We are moving to a new naming convention in order to use different docker images for different purposes. What was the `code-dot-org` image will become the `cdo-drone` image. For now, these docs advise publishing both names.
+
 Tag the new image with an incremented version number. You can get the image id from the end of the build log or from `docker image list`.
 
 ```
 docker tag <image_id> codedotorg/code-dot-org:<version>
+docker tag <image_id> codedotorg/cdo-drone:<version>
```

Comment:
We don't define what are versioning scheme is. I'd suggest a simple incrementing integer, or a date based scheme.